src/components/BaseLightBoxProfileEdit.vue

```diff
@@ -33,7 +33,7 @@
     <div class="profile-edit__aside">
       <div class="portrait">
         <div class="portrait__container" @click="profileEditorUploadThumbnail">
-          <img class="portrait__thumbnail" :src="thumbnail" alt="thumbnail">
+          <img class="portrait__thumbnail" v-show="thumbnail" :src="`http://www.readr.tw/${thumbnail}`" alt="thumbnail">
```

Comment:
should we get the hostname from the config file?

Reply:
move hostname to config by next commit